### PR TITLE
Fix a crash when a user clicks quick menu button

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch fixes a crash when a user clicks quick menu button. To
resolve this issue, add null check for `@Nullable` parameter.

Refs: #1013

### CODE REVIEW

**Review by romandev — APPROVED** [review]
lgtm

File: src/org/triple/banana/quick_menu/QuickMenuMiddleButton.java [hunk 1]
```diff
@@ -47,6 +47,9 @@ public void setCompoundDrawablesWithIntrinsicBounds(@Nullable Drawable left,
             @Nullable Drawable top, @Nullable Drawable right, @Nullable Drawable bottom) {
         Drawable background = ResourcesCompat.getDrawable(getResources(),
                 R.drawable.quick_menu_button_image_background, getContext().getTheme());
+        if (top == null || background == null) {
+            return;
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuMiddleButton.java:51]: I think we should call `super.setCompoundDrawablesWithIntrinsicBounds()` here

**Review by romandev — CHANGES_REQUESTED** [review]

File: src/org/triple/banana/quick_menu/QuickMenuMiddleButton.java (hunk 1 above)
Comment (jdragonbae) [reply src/org/triple/banana/quick_menu/QuickMenuMiddleButton.java:52]: Ack. Thanks for the review.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**